.travis.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
I don't know what's best. Following the warnings triggered by your tool and write strictly valid YAML or just following how the Symfony community writes YAML.